### ISSUE TITLE
change hostname of bot

### ISSUE THREAD

**Issue by ianremsen** [langel/botb_irc_bot#25]
- BotB2 (~nodebot@66-90-166-83.dyn.grandenetworks.net) has joined

needs to become
- BotB (~botb@66-90-166-83.dyn.grandenetworks.net) has joined

**Comment by langel** [comment]
don't know why this autoclosed
this is the final ticket for the ready to replace milestone

**Comment by jangler** [comment]
It autoclosed because of the "resolves #25" in that commit message for 92bf98b. I can stop putting those notes in my commit messages if you like!

**Comment by langel** [comment]
No, keep doing that.  It's just for this ticket.  This ticket is an EPIC LANDMARK! ;P
